hosts.yml

```diff
@@ -19,8 +19,17 @@ ssl_CA:
 zookeeper:
   hosts:
     ip-172-31-24-74.us-east-2.compute.internal:
+      zookeeper:
+        config:
```

Comment:
By making this part of the _config_ dict, it is also added to the Zookeeper config file itself (by the loop at the top of server.properties.j2), which we should avoid. Maybe just have this directly under zookeeper?